src/util.c

```diff
@@ -1705,7 +1705,7 @@ nats_Base32_Init(void)
     int         i;
 
     for (i=0; i<(int)sizeof(base32DecodeMap); i++)
-        base32DecodeMap[i] = (char) 0xFF;
+        base32DecodeMap[i] = '\xFF';
```

Comment:
beats me why this was warning on Win32, but it was (int const truncation). Still doing it in `nkeys.c` but I was unable to fix there without digging into the implementation.